recipes.json

```diff
@@ -0,0 +1,332 @@
+          "rating": 5,
+          "comment": "With the garnish, the dish looks so yumm!!"
+        },
+        {
+          "username": "Harish",
+          "rating": 4,
+          "comment": "Great taste! Would recommend to all!!"
+        }
+      ]
+    },
+    "tags": ["40 mins", "1 serving", "Intermediate"]
+  },
+  {
+    "id": "4",
+    "name": "Chocolate Raspberry Mousse",
+    "image": "images/recipiepage/chocolate mousse.jpg",
+    "description": "Dazzle the senses with this indulgently creamy mousse. It has a beautifully rich texture that is balanced by tart and juicy fresh raspberries. The caramel shard topping is not only beautiful, but also adds a satisfying crunch to this dessert.",
+    "time": "20 mins",
+    "servings": 1,
+    "difficulty": "Easy",
+    "ingredients": {
+      "chocolate-mousse": [
+        "1.8 oz (50 g) silken tofu, drained",
+        "1.2 oz (33 g) melted dairy-free dark chocolate",
+        "1 tsp (5 g) granulated sugar",
+        "1 pinch (0.3 mL) pure vanilla extract",
+        "1 pinch (0.1 g) ground cardamom (optional)"
+      ],
+      "fruit-layers": [
+        "4 Tbsp (31 g) fresh raspberries, for mashing",
+        "⅓ cup (41 g) fresh raspberries, remaining whole"
+      ],
+      "whip": [
+        "1 Tbsp + 1 tsp (20 mL) vegan whipping cream",
+        "1 tsp (5 g) granulated sugar",
+        "½ tsp (1 g) cream stabilizer"
+      ],
+      "optional-garnishes": ["chocolate shavings", "caramel shards"]
+    },
+    "instructions": [
+      "1. Add the chocolate mousse ingredients to a food processor. Blend until smooth.",
+      "2. To each of the serving glasses, add ¼ cup of raspberries. Lightly mash the raspberries with the back of a spoon or tamper.",
+      "3. Reserve about ¼ of the blended chocolate mixture in a bowl. Divide the remaining between your glasses over the mashed raspberries. Place the glasses in the fridge while preparing the next layer.",
+      "4. Add all of the whip ingredients to a large bowl. Whip with an electric hand mixer until stiff peaks form, about 7 minutes. Reserve some of the whipped cream as a topping if desired.",
+      "5. Then add the reserved blended chocolate mixture to the remaining whipped cream and gently fold together.",
+      "6. Remove the cups from the fridge and add some whole raspberries as the next layer.",
+      "7. Then divide this whipped chocolate mousse between the glasses.",
+      "8. Top with the reserved whipped cream and any desired garnishes. Enjoy!"
+    ],
+    "reviews": {
+      "average_rating": 4.7,
+      "user_reviews": [
+        {
+          "username": "Saran Hiruthik",
+          "rating": 5,
+          "comment": "Absolutely delicious! So moist and rich, the best vegan cake I've had."
+        },
+        {
+          "username": "Kavinesh",
+          "rating": 4,
+          "comment": "Very tasty, but I would reduce the sugar next time."
+        }
+      ]
+    },
+    "tags": ["20 mins", "1 serving", "Easy"]
+  },
```

Comment:
This LGTM! Just a suggestion though: it's better to store the unique usernames in lowercase without spaces for consistency, and you could also add a timestamp field to track when the review was posted.

Reply:
one doubt

Reply:
Also, you could remove the avg_rating field, as it can be calculated dynamically based on the individual user ratings

Reply:
> will backend be a addition to this project

Can be done 🫡 Right now, I wanted to clean up the frontend and making it dynamic. The repo is currently a bit of a mess since it’s mostly pure HTML and CSS, which is why I wanted to store the data as JSON for now. You can create an issue to work on the backend, and we can integrate it later on

Reply:
okayy. got it!! 🫡